docs/source/yiip_example.rst

```diff
@@ -0,0 +1,31 @@
+YiiP Protein Example
+====================
+
+To get started immediately with *Zarrtraj*, we have made the topology and trajectory of the 
+`YiiP protein in a POPC membrane <https://www.mdanalysis.org/MDAnalysisData/yiip_equilibrium.html>`_
+publicly available for streaming. The trajectory is stored in in the `zarrmd` format 
+for optimal streaming performance. 
+
+To access the trajectory, follow this example:
+
+.. code-block:: python 
+
+    import zarrtraj
+    import MDAnalysis as mda
+    import fsspec
+
+
+    with fsspec.open("gcs://zarrtraj-test-data/YiiP_system.pdb", "r") as top:
+
+        u = mda.Universe(
+            top, "gcs://zarrtraj-test-data/yiip.zarrmd", topology_format="PDB"
```

Comment:
I get an error running it: `KeyError: 'content-type'`

Reply:
Thank you for testing! I replied in the discord, better for debugging

Reply:
We resolved this.